### PR TITLE
Cdsid 580 825

### PR DESCRIPTION
**Pull request by russellthorn**

### CODE REVIEW

File: app/views/goods_items_previousdocs.scala.html [hunk 1]
```diff
@@ -30,36 +30,41 @@
...
-        )(previousDocuments.map(a => (a.categoryCode.getOrElse(""),
-        a.id.getOrElse(""),
-        a.typeCode.getOrElse(""),
-        a.lineNumeric.getOrElse("")))))
-
+    <fieldset>
+        <legend>
+            <h1 class="heading-large">@messages("addPreviousDocument.goodsitem.title")</h1>
+        </legend>
+        @if(previousDocuments.isEmpty) {
```
Comment (kurtlogan) [inline app/views/goods_items_previousdocs.scala.html]: Empty if statement isn't needed

Comment (russellthorn) [reply app/views/goods_items_previousdocs.scala.html]: Oops maybe that should be `@if(!previousDocuments.isEmpty) {`

File: app/views/goods_items_origins.scala.html [hunk 2]
```diff
@@ -37,28 +37,36 @@
...
+                                   origins.map(
+                                       a => (a.countryCode.getOrElse(""),
+                                       a.typeCode.getOrElse("")))), Some(tableHeader))
+                       </div>
+                   }
+           
+           @if(origins.isEmpty) {
+                   @components.input_text(field = form("countryCode"),label = messages("origin.countryCode"))
+                   @components.input_text(field = form("typeCode"),label = messages("origin.typeCode"))
+               </fieldset>
```
[the remark this answers is not in the document]
Comment (russellthorn) [reply app/views/goods_items_origins.scala.html]: Cool, will change!

File: app/views/goods_items_previousdocs.scala.html (hunk 1 above)
Comment (russellthorn) [reply app/views/goods_items_previousdocs.scala.html]: Oh, I didn't change that.... did I?

**Pull request merged**